changelogs/fragments/54490-docker_swarm-return-unlock-key.yaml

```diff
@@ -1,2 +1,3 @@
 minor_changes:
   - "docker_swarm - ``UnlockKey`` will now be returned when ``autolock_managers`` is ``true``."
+  - "docker_swarm_info - Add option ``unlock_key`` to retrieve the swarm unlock key."
```

Comment:
`docker_swarm_info` has been added for Ansible 2.8, so it does not need a changelog fragment.

Reply:
I missed that. Fixed.